### DISCUSSION TITLE
[FEATURE] The ability to view tables side by side

### DISCUSSION THREAD

**Discussion by BDR159** [sqlitebrowser/sqlitebrowser#3285] [Ideas]
The ability to view tables side by side would mean I wouldn't have to constantly switch between the parent table and the table that contains a foreign key attribute that references it. I constantly find myself going to the parent table, trying to remember the ID and the information associated with it, and then quickly switch to the child table to insert foreign key before I forget what it represents.

**Comment by justinclift** [comment]
Oh, I *think* you're in luck.  Our nightly builds have had the ability to open multiple "Browse Data" tabs for a while.

If you're using macOS or Windows, the nightlies are here:

&nbsp; &nbsp; https://nightlies.sqlitebrowser.org/latest/

If you're using Linux, then there a several options available too (eg snap packages, AppImage's, etc).

Maybe try that out? :smile:

**Reply by BDR159** [reply]
Yep, thank you very much. Due to my poor updating habits, I am using the intel version instead of the arm64 version as that version requires macOS 12.6. But I'm willing to tolerate a slight performance hit due to Rosetta.

The nightly build seems to have some other great features as well!

**Reply by justinclift** [reply]
Ahhh.  Hopefully it works decently.  We've had the occasional crash report from people using intel builds through Rosetta, which was a big part of the reason we have arm64 builds available now too.

If our Intel nightly build does seem to flake out on you a bit too often, then compiling our application from source on macOS is generally pretty straight forward:

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser/blob/master/BUILDING.md#building-an-app-bundle

It's been a while since I've gone through those steps manually on macOS personally, but no-one's complained they don't work any more / need updating.  So, this might be a useful option. :smile:
